_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: newsletter
+title: "Newsletter #5 - TODO"
+category: newsletter
+permalink: /news/2015/april/
+---
+
+Welcome to the fifth newsletter for Neovim.
+
+## General News
+
+### Blackduck 2014 Open Source Rookie
+
+[Blackduck][blackduck-site], a company that helps other corporations in the
+adoption of open source software, chose Neovim as one of the projects to feature
+of this last year. The company annually recognizes some of the top new open
+source projects that started during the year.
+
+### ICCF Holland Donations
+
+Mentioned in the last letter, [Unixstickers][iccf-unixstickers] has added
+[Neovim stickers][iccf-stickers] to their inventory. Every sticker that is
+bought increases the amount donated to [ICCF Holland][iccf-site], Bram
+Moolenaar's charity.
+
+Since they've been added, $106 has been raised which Unixstickers [just
+donated][iccf-tweet] on March 24th.
+
+### Related Projects
+
+The [Neovim Wiki][related-main-wiki] contains a page titled [Related
+projects][related-wiki] that contains a list of current GUI projects, API
+clients, plugins that take advantage of Neovim features, and projects with
+direct support for Neovim.
+
+Some of the projects might be experimental which means there might be bugs and
+the projects might still be in progress.
+
+Do you know of another project that can be listed? Don't be afraid to add a link
+yourself!
+
+## Development News
+
+### Replacing Terminal UI
+
+TODO
+
+### Builtin Terminal Emulation
+
+@tarruda has finished [his pull request][term-pr] adding a new `:terminal`
+command which when ran launches a full terminal emulator. The terminal is using
+Neovim's native buffers and windows which means the user can enter [normal
+mode][term-normal] by pressing `<C-\><C-n>`.
```

Comment:
Now that it's been documented, perhaps you could append something like the following to this paragraph:

```
See [`:h nvim-terminal-emulator`][help-terminal-emulator] for more information.

...

[help-terminal-emulator]: http://neovim.org/doc/user/nvim_terminal_emulator.html#nvim-terminal-emulator
```